### PR TITLE
adds meal model migration and meal name validation

### LINKED ISSUE #12: Meals Table Initial Schema
A Meal resource should be stored in the database on a Meals table.
The structure of a Meal resource is as follows:
```
ID: Integer
Name: String
```

### PR DESCRIPTION
**Pull request by tnodland**
resolves #12

### CODE REVIEW

File: migrations/20190824182911-create-meal.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Meals', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
```
Comment (BrennanAyers) [inline migrations/20190824182911-create-meal.js]: Let's add the `allowNull` here as well just to make sure everything works if the other file is missing? @tnodland

Comment (tnodland) [reply migrations/20190824182911-create-meal.js]: I'll add both to the next PR!

Comment (tnodland) [reply migrations/20190824182911-create-meal.js]: By both I mean meals and foods

Comment (BrennanAyers) [reply migrations/20190824182911-create-meal.js]: 👍

**Pull request merged**